src/core/services/interimElement/interimElement.spec.js

```diff
@@ -263,6 +263,8 @@ describe('$$interimElement service', function() {
 
         Service = $$interimElement();
 
+        ieShow = Service.show;
```

Comment:
I agree, we may want to refactor it and use something like this; and I'll definitely do this in future tests.